### PR TITLE
Work for splitting #2370 rabbitmq

### PR DESCRIPTION
**Pull request by kz0817**
[hap2][RabbitMQConnector] Retry on the failure of publish.
    
This patch adds a feature that retries to publish again on a failure.

### CODE REVIEW

File: server/hap2/hatohol/rabbitmqconnector.py [hunk 1]
```diff
@@ -134,15 +147,23 @@ def __publish(self, msg):
...
+                            "exchange": "",
+                            "routing_key": self._queue_name,
+                            "body": msg,
+                            "properties": pika.BasicProperties(
+                                        content_type="application/json")
+                            }
+        try:
+            self._channel.basic_publish(**publish_argument)
+        except exceptions.ConnectionClosed:
+            self._connect()
```
Comment (cosmo0920) [inline server/hap2/hatohol/rabbitmqconnector.py]: It seems that this retrying feature causes retrying when failing to publish forever.
Should we add upper limit of retrying count?
Or, this patch is enough for retrying feature for now?

Comment (masa0612) [inline server/hap2/hatohol/rabbitmqconnector.py]: > It seems that this retrying feature causes retrying when failing to publish forever.

No such thing.
This sentence retry once.

Comment (cosmo0920) [inline server/hap2/hatohol/rabbitmqconnector.py]: Hmm..., I see.

**Comment by cosmo0920** [comment]
My concern is above comment.
But it is wrong according to @masa0612 .
Let's merge this PR!
:shipit:

**Pull request merged**